apps/cms/src/admin/views/MerchSales.tsx

```diff
@@ -5,19 +5,28 @@ import ViewTemplate from "./ViewTemplate";
 import { Column } from "payload/dist/admin/components/elements/Table/types";
 import { Order } from "../../@types/Order";
 import OrdersApi from "../../apis/orders.api";
-import { IOrder } from "../../@types/IOrder";
 import { RenderCellFactory } from "../utils/RenderCellFactory";
 import SortedColumn from "../utils/SortedColumn";
 import { Table } from "payload/dist/admin/components/elements/Table";
+import { useHistory } from 'react-router-dom';
 
 
 const MerchSales: AdminViewComponent = ({ user, canAccessAdmin }) => {
     // Get data from API
-    const [data, setData] = useState<IOrder[]>(null);
-    useEffect(() => {
-      OrdersApi.getOrders()
-        .then((res: IOrder[]) => setData(res))
-        .catch((error) => console.log(error));
+  const [data, setData] = useState<Order[]>(null);
+  const history = useHistory();
+  useEffect(() => {
+    const fetchOrders = async () => { 
+      try {
+        const orders: Order[] = await OrdersApi.getOrders();
+        setData(orders);
+      } catch (error) {
+        console.error(error);
+        setData([]);
```

Comment:
Can we do better error handling than this? Maybe even just displaying an error message?